python/cuml/fil/fil.pyx

```diff
@@ -137,6 +137,12 @@ cdef class TreeliteModel():
         model.set_handle(handle)
         return model
 
+    @staticmethod
```

Comment:
Not sure if having a static method to do that is the best. Wouldn't it be better to add a parameter to `__cinit__` with the default value of `NULL`?

Reply:
I think this is consistent with the `from_filename` static method